File: src/1848.minimum-distance-to-the-target-element.1975/solution.ts
function getMinDistance (nums: number[], target: number, start: number): number {
    let result = nums.length;
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] !== target) {
            continue;
        }
        result = Math.min(result, Math.abs(i - start));
    }
    return result;
}

File: src/1848.minimum-distance-to-the-target-element.1975/solution.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        int result = nums.length;
        for(int i=0;i<nums.length;i++){
            if(nums[i] != target){
                continue;
            }
            result = Math.min(result,Math.abs(i-start));
        }
        return result;
    }
}